scene/3d/baked_lightmap.cpp

```diff
@@ -449,7 +449,7 @@ int32_t BakedLightmap::_compute_bsp_tree(const Vector<Vector3> &p_points, const
 		ERR_FAIL_COND_V(p_simplex_indices.size() <= 1, 0); //should not happen, this is a bug
 
 		// Failed to separate the tetrahedrons using planes
-		// this means Delaunay borked at some point.
+		// this means Delaunay broke at some point.
```

Comment:
FYI :) https://www.urbandictionary.com/define.php?term=borked

But yeah, let's change it and silence warnings from spellchecker tools.

Reply:
oh, good to know